odoo/fields.py

```diff
@@ -785,6 +826,11 @@ def convert_to_record(self, value, record):
         """
         return False if value is None else value
 
+    def convert_to_record_multi(self, values, record):
```

Comment:
Make the API more consistent: `record` is not a single record
```suggestion
    def convert_to_record_multi(self, values, records):
```